docs/dev/testnet/important-links.md

```diff
@@ -13,6 +13,8 @@
 - Network Name: `zkSync alpha testnet`
 - New RPC URL: `https://zksync2-testnet.zksync.dev`
 - Chain ID: `280`
+- Currency Symbol(Optional): `ETH`
```

Comment:
mmm...... not sure what's best here: we don't really have a native currency, but I understand MM wants to display something